meteor/client/lib/DocumentTitleProvider.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react'
+
+import { translateWithTracker, Translated } from './ReactMeteorData/ReactMeteorData'
+import { CoreSystem, ICoreSystem } from '../../lib/collections/CoreSystem'
+
+import { ReactiveVar } from 'meteor/reactive-var'
+
+export const documentTitle = new ReactiveVar<null | string>(null)
+
+interface IProps {}
+
+interface ITrackedProps {
+	cs: ICoreSystem | undefined
+	doc: string | null
+}
+
+export const DocumentTitleProvider = translateWithTracker((props: IProps) => {
```

Comment:
This should be documented, preferably with an example of usage (`documentTitle.set('The title of the page')`).

Reply:
I've added some docs to the documentTitle ReactiveVar & it shows up in Intellisense.